pkg/aws/aws.go

```diff
@@ -68,6 +67,19 @@ func (a *awsSvc) getRegions() error {
 	return nil
 }
 
+func (a *awsSvc) createEC2Service(region string) *ec2.EC2 {
+	var baseConfig *aws.Config
+	roleArnName := os.Getenv("ROLE_ARN")
```

Comment:
Does it make sense to grab this within `New()` and store it within the struct? Then you're not constantly hitting `os.Getenv()` every time you need a new `ec2.EC2`, not that it's slow but it's work that can be done once and reused.

Reply:
I put them as comments in the code in case something comes up.